ipatests/prci_definitions/nightly_latest.yaml

```diff
@@ -1265,6 +1265,18 @@ jobs:
         timeout: 3600
         topology: *master_1repl
 
+  fedora-latest/test_ipahealthcheck_TestIpaHealthCheck:
+    requires: [fedora-latest/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-latest/build_url}'
+        test_suite: test_integration/test_ipahealthcheck.py::TestIpaHealthCheck
```

Comment:
As test_ipahealthcheck.py currently contains a single test class, you can also define `test_suite: test_integration/test_ipahealthcheck.py`. This way, if the py file is later extended with another test class you won't need to modify the job definition.
Up to you, but if you implement my suggestion, change the test name into `fedora-latest/test_ipahealthcheck`.

Reply:
@flo-renaud added the above mentioned changes.